bitnami/mongodb-sharded/values-production.yaml

```diff
@@ -209,6 +209,12 @@ shardsvr:
       ## Use 0 to disable
       ##
       maxUnavailable: 1
+    ## Name of a Service Account to be used by shardsvr data pods
+    ##
+    serviceAccountName:
+    ## Whether to create a Service Account for shardsvr data pods automatically
+    ##
+    createServiceAccount: false
```

Comment:
In other charts we're using the approach below:
```yaml
## K8s Service Account.
## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
##
serviceAccount:
  ## Specifies whether a ServiceAccount should be created.
  ##
  create: true
  ## The name of the ServiceAccount to use.
  ## If not set and create is true, a name is generated using the XXX.fullname template
  ##
  # name:
```
We should follow the same structure on this case too.

Reply:
Will fix!